csharp-mode.el

```diff
@@ -3067,10 +3077,10 @@ Key bindings:
 
   ;; The paragraph-separate variable was getting stomped by
   ;; other hooks, so it must reside here.
-  (setq paragraph-separate
+  (set (make-local-variable paragraph-separate)
         "[ \t]*\\(//+\\|\\**\\)\\([ \t]+\\|[ \t]+<.+?>\\)$\\|^\f")
 
-  (setq beginning-of-defun-function 'csharp-move-back-to-beginning-of-defun)
+  (set (make-local-variable beginning-of-defun-function) 'csharp-move-back-to-beginning-of-defun)
```

Comment:
Same for this one.

Reply:
I tried to be consistent with for instance `(set (make-local-variable 'comment-auto-fill-only-comments) ...`; will do.

Reply:
It may be consistent, but it's no longer needed, so I say lets write new code using the new constructs :)